tools/sgen/input/FluentUIStyle.yml

```diff
@@ -296,3 +303,10 @@ AP_MSFDrawerTokens:
   shadow2Blur: $Shadow.shadow28.blur2
   shadow2OffsetX: $Shadow.shadow28.x2
   shadow2OffsetY: $Shadow.shadow28.y2
+  minMargin: [horizontal: 44pt, vertical: 20pt]
+  cornerRadius: 14pt
```

Comment:
Shouldn't this be mapped to one of the Border radius tokens?

Reply:
`Border.radius` doesn't have a mapped value in design spec. I can introduce one but that would create more confusion if designers can't trace it back. I have kept constants value for now instead of resolving this discrepancy. 

To resolve this discrepancy we can either: 
1. Add new constants to extend design specs (by using special nomenclature small, medium, custom1, custom2 .. etc]
2. Map to closest available constant. 
```yaml
  cornerRadius: $Border.radius.xlarge
```

Any thoughts?

Reply:
I have mapped this to core tokens instead of using constants.